src/collective/nitf/tests/test_content.robot

```diff
@@ -23,9 +23,9 @@ Test Folder Full View
     Open Add New Menu
     Click Link  css=a#folder
     Page Should Contain  Add Folder
-    Input Text  css=#form-widgets-IDublinCore-title  Test Folder
+    Input Text  css=#form-widgets-IDublinCore-title,#title  Test Folder
```

Comment:
AFAIK RF accepts all CSS 1 and CSS 2 selectors http://www.w3schools.com/cssref/css_selectors.asp

I vaguely remember to have problems with some Pseudo elements and Pseudo classes https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors